src/context.h

```diff
@@ -27,6 +27,18 @@
 #include "Poco/Timestamp.h"
 #include "Poco/Util/Timer.h"
 
+#ifdef TOGGL_ALLOW_UPDATE_CHECK
+# define UPDATE_CHECK_DISABLED false
+#else
+# define UPDATE_CHECK_DISABLED true
+#endif
+
+#if defined(TOGGL_PRODUCTION_BUILD) && !defined(APP_ENVIRONMENT)
```

Comment:
I don't think testing `!defined(APP_ENVIRONMENT)` here is necessary. if `TOGGL_PRODUCTION_BUILD` is defined it is always "production" otherwise it's "development"

Reply:
I'm not sure I follow. Overriding what exactly, APP_ENVIRONMENT? Can you elaborate on a scenario when we would use it so I get a better idea?

Reply:
Well, having this in place doesn't really break anything, it just gives an opportunity to override the `production` / `development` choice by hand if necessary in the future, for instance if the servers get renamed, if we add a third one or in some other scenario.

Reply:
all good, let's keep it.